### ISSUE TITLE
Audit the "Updating" aria live message

### ISSUE THREAD

**Issue by afercia** [obenland/shiny-updates#143]
As part of the a11y review, see #129 

Not sure the "updating" message ('Updating... please wait.') should be used for the button aria-label. Yes, currently, the plugin cards change the update button aria label but they use 'updatingLabel'     => __( 'Updating %s...' ) referencing the plugin name.

Instead, 'Updating... please wait.' should be used for `wp.a11y.speak()`. 

Seems only Themes updates dispatch the "updating" message to the live region. It should be done for all update types (core, themes, plugins, translations), on all screens (update-core.php and the other screens).

**Comment by swissspidy** [comment]
Let's use `Updating translations…`  and `Updating WordPress…` in addition to `Updating %s...` for plugins and themes. Sounds good?

**Comment by swissspidy** [comment]
Oh, and perhaps `Installing all updates…` or `Updating site…` for 'Update All'?
